tests/test_processors.py

```diff
@@ -321,3 +323,43 @@ def aggregateRating(self):
     assert page.aggregateRating == AggregateRating(
         ratingValue=3.8, bestRating=10, reviewCount=5
     )
+
+
+@pytest.mark.parametrize(
+    "input_value,expected_value",
+    [
+        (None, None),
+        ("", ""),
+        ("Value ", "Value"),
+        (" Value", "Value"),
+        (" Value ", "Value"),
```

Comment:
`("foo bar", "foo bar"),` (i.e. make sure we do not remove spaces between words)